### DISCUSSION TITLE
How can I make the player lose the land depending on the size of the land and as the land is bigger, it takes longer to lose?

### DISCUSSION THREAD

**Discussion by thiagogebrimm** [GriefPrevention/GriefPrevention#1568] [Q&A]
I don't know if there's how, but I've been on servers that use griefprev and it has this function

**Comment by Jikoo** [comment]
They're probably running [GPClaimExpiration](https://github.com/GPAddons/GPClaimExpiration). Builds can be acquired off [the CI](https://ci.appveyor.com/project/Jikoo/gpclaimexpiration/build/artifacts).
